.github/workflows/examples-tfplan-tests.yml

```diff
@@ -1,12 +1,10 @@
 name: e2e-all-examples-parallel
```

Comment:
Arn't we using this workflow now [e2e-parallel-full.yml](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/main/.github/workflows/e2e-parallel-full.yml)?
Should we remove this [examples-tfplan-tests.yml](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/pull/390/files#diff-186b79d57acb8e30733d94bdd38cc5fc0f0b1f42bc34653a123354964d557730)?

Reply:
I believe the team wanted to keep TF plan only workflow for now.
We can think about having it with the new e2e-parallel, with some flag to say "Run only until e2e plan", so we can reduce the number of workflows, wdyt?

Reply:
fine with me